src/data/mystery-encounters/encounters/clowning-around-encounter.ts

```diff
@@ -127,6 +127,13 @@ export const ClowningAroundEncounter: MysteryEncounter =
       encounter.setDialogueToken("ability", new Ability(ability, 3).name);
       encounter.misc = { ability };
 
+      // Decide the random types for Blacephalon. They should not be the same.
+      const firstType: number = randSeedInt(18);
+      let secondType: number = randSeedInt(17);
+      if ( secondType >= firstType ) {
+        secondType++;
+      }
```

Comment:
This makes it so the second type can never be Fairy unless both roll Dark. I'd go with this instead:
```suggestion
      let secondType: number = randSeedInt(18);
      while (secondType === firstType) {
        secondType = randSeedInt(18);
      }
```